src/React/Redux.purs

```diff
@@ -169,16 +169,18 @@ foreign import data REDUX :: !
 
 foreign import data Store :: * -> * -> *
 
-foreign import data ReduxReactClass :: * -> * -> * -> *
+foreign import data ReduxReactClass :: * -> * -> * -> * -> *
 
-foreign import connect_ :: forall state props props'. Fn3 (state -> props -> Tuple state props) (Tuple state props -> props') (React.ReactClass props') (ReduxReactClass state props props')
+foreign import connect_ :: forall state props props' action. Fn3 (state -> props -> Tuple state props) (Tuple state props -> props') (React.ReactClass props') (ReduxReactClass props props' state action)
 
 foreign import dispatch_ :: forall eff action props state. Fn2 (React.ReactThis props state) action (Eff (ReduxEffect eff) action)
```

Comment:
Should the order of the type parameters be `eff props state action`? This comment applies generally. There are something other functions where `action` is still before `state`. Not sure if we decided to reorder these too or not.